### ISSUE TITLE
Meta: 🎰 SEO

### ISSUE THREAD

**Issue by sarahsporck** [digitalfabrik/integreat-app#2385]
As a developer of a website I want a higher ranking in the search results such that more users visit my webpage.

## Tasks:

- [ ] #2132 
- [ ] #2241 
- [ ] #1284 
- [ ] #1283 
- [ ] #1219 
- [ ] #1218 
- [ ] #1018

**Comment by sarahsporck** [comment]
[**@steffenkleinle**](https://github.com/steffenkleinle) commented on [Aug 29, 2023, 2:35 PM GMT+2](https://github.com/digitalfabrik/integreat-app/issues/2385#issuecomment-1697360468 "2023-08-29T12:35:46Z - Replied by Github Reply Comments"):
> Yeah maybe, but all tasks in this meta issue are done anyway, so I don't see the need for this meta issue. But maybe I am also misunderstanding something here.

I think this calls for a definition of done here. So when I look at accessibility (which must yet be converted to a meta issue) I'm wondering when it can be considered as done, as there will probably always be some issues that come up regarding accessibility. Should we only define Meta Issues for "completable" tasks? What do we do in other cases? I see what you mean here, and also agree with you. This ticket can surely be closed. But what if we have another issue regarding seo. Is it then separated from the other Meta issues? 

In my opinion, these types of issues should have a measurable state of completion. If we say this ticket contains the whole SEO user story, then we can close it. All new issues related to SEO (until django is ready) should only be bugs and features not related to the whole SEO setup. Long story short. I'll close this ticket. I think I came to my conclusion, at least in the process of writing this ^^. But the question what to do about (non-completable) "Milestones" remains.

**Comment by steffenkleinle** [comment]
I agree and think all meta issues AND milestones should be completable. There should only be a meta issue for Accessibility or SEO if we have some new big topic in that regard. Examples would imo be: We did not have any SEO/meta tags at all and want to introduce SEO which requires separate issues to be completed (probably like this was meant in the beginning). Same for Accessibility, I think it makes sense to have a meta issue here because we want to apply for funding and then do those issues in that context. Once the funding is done and we finished all issues in the meta issue, the meta issue should be closed.

Issues that are opened afterwards should not be added to the old meta issues anymore and are instead just standalone issues like any other issue. If we want to mark that it is regarding SEO or Accessibility, we should use labels for that. But in general I don't think we would need to specify a label/meta issue for just some task/bug just because it is SEO or accessibility, I don't think that those issues are in any way special. Otherwise we would probably also need meta issues or labels for e.g. events, RTL, offline, ....
